keyboards/mitosis/rules.mk

```diff
@@ -27,6 +27,7 @@ NKRO_ENABLE = yes		# USB Nkey Rollover - not yet supported in LUFA
 # MIDI_ENABLE = yes 		# MIDI controls
 UNICODE_ENABLE = yes 		# Unicode
 # BLUETOOTH_ENABLE = yes # Enable Bluetooth with the Adafruit EZ-Key HID
+LTO_ENABLE = yes # Without LTO, VIA firmware size is very close to the limit.
```

Comment:
This should go in the keymap's rules.mk, rather than changing it at the keyboard level. 

```suggestion
```